modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -1010,7 +1010,13 @@ public abstract class Base${schemaName}ResourceImpl
 								try {
 						</#if>
 
-						${deleteByIdJavaMethodSignature.methodName}(${schemaVarName}.${getterMethodName}());
+						${deleteByIdJavaMethodSignature.methodName}(${schemaVarName}.${getterMethodName}()
+							<#list deleteByIdJavaMethodSignature.javaMethodParameters as javaMethodParameter>
```

Comment:
Please, check the alignment. I remember the last PR about REST Builder with the same misalignment